backend/tournesol/migrations/0023_make_new_entity_fields_notnull.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.11 on 2022-02-03 14:29
```

Comment:
Any reason to create 5 migration files? If not, you could simply concatenate the list of operations.

Reply:
There is indeed no reason to keep all migrations files. I'll regroup them in coherent operations.